examples/lighting-app/silabs/src/AppTask.cpp

```diff
@@ -54,7 +56,7 @@ using namespace ::chip::DeviceLayer;
 using namespace ::chip::DeviceLayer::Silabs;
 
 namespace {
-LEDWidget sLightLED;
+RGBLEDWidget sLightLED;
```

Comment:
how would that work for board that do not support rgb?

Reply:
Removed

Reply:
Added condition so that it will work for non-rgb boards also ` (defined(SL_MATTER_RGB_LED_ENABLED) && SL_MATTER_RGB_LED_ENABLED == 1)`